deps/wazuh_testing/wazuh_testing/tools/agent_simulator.py

```diff
@@ -199,7 +209,7 @@ def register(self):
         self.key = registration_info[3]
         ssl_socket.close()
         sock.close()
-        print("Registration - {}({})".format(self.name, self.id))
+        logging.info("Registration - {}({})".format(self.name, self.id))
```

Comment:
I would use `logging.debug()` for all debug messages, because all these messages will be printed even if debug of the agent simulator is set to false